frontend/frontend-sdk/src/lib/Dto.ts

```diff
@@ -268,6 +268,7 @@ export interface Claims {
   email?: Pick<Email, "address" | "is_primary" | "is_verified">;
   username?: string;
   session_id: string;
+  [key: string]: unknown;
```

Comment:
Do you see a way to improve this one, so that the user can provide a type for the custom claims?